### PR TITLE
Validation: description-missing fixes.

### PR DESCRIPTION
**Pull request by brittmsantos**
Added missing descriptions to fix validation errors.

### CODE REVIEW

**Comment by brittmsantos** [comment]
@stevemunk @rbrundritt

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

File: BingMaps/v8-web-control/map-control-concepts/drawing-tools-module-examples/get-shapes-from-drawingmanager.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Get Shapes from DrawingManager | Microsoft Docs"
+description: Provides a code example that shows how to get all shapes that are in the drawing manager at any time.
```
Comment (stevemunk) [inline BingMaps/v8-web-control/map-control-concepts/drawing-tools-module-examples/get-shapes-from-drawingmanager.md:3]:
```suggestion
description: Provides a code example that shows how to get all shapes that are in the drawing manager at a given time.
```

File: BingMaps/v8-web-control/map-control-concepts/geojson-module-examples/read-geojson-using-html5-filereader.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Read GeoJSON using HTML5 FileReader | Microsoft Docs"
+description: Provides a code example that shows how to enable drag and drop a local GeoJSON file onto a map using the FileReader and Drag & Drop HTML5 APIs.
```
Comment (stevemunk) [inline BingMaps/v8-web-control/map-control-concepts/geojson-module-examples/read-geojson-using-html5-filereader.md:3]:
```suggestion
description: Provides a code example that shows how to enable drag and drop a local GeoJSON file onto a map using the FileReader and Drag & Drop APIs available in HTML5.
```

File: BingMaps/v8-web-control/map-control-concepts/geojson-module-examples/read-local-geojson-object-example.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Read Local GeoJSON Object Example | Microsoft Docs"
+description: Provides a code example that shows how to parse a locally defined GeoJSON object into a Bing Maps shape using the GeoJSON module.
```
Comment (stevemunk) [inline BingMaps/v8-web-control/map-control-concepts/geojson-module-examples/read-local-geojson-object-example.md:3]:
```suggestion
description: Provides a code example that shows how to parse a locally defined GeoJSON object into a Bing Maps shape using the GeoJSON module. then adds it to the map.
```

**Review by stevemunk — CHANGES_REQUESTED** [review]

**Comment by brittmsantos** [comment]
@stevemunk 

I've implemented the requested changes. Let me know if there are any other issues preventing approval and merge. Thanks!

**Pull request merged**